src/content/services/frameManager.ts

```diff
@@ -176,21 +176,22 @@ export class FrameManager {
     this.observeDocumentForFrames();
   }
 
+  static handledMessages: ContentToContentMessage['message'][] = [
+    'INITIALIZE_IFRAME',
+    'IS_MONETIZATION_ALLOWED_ON_START',
+    'IS_MONETIZATION_ALLOWED_ON_RESUME',
+  ];
```

Comment:
What is the reasoning behind making them `static`?

Reply:
Such constants make more sense as statics, as they are not depending on anything specific to this class (like class methods too). Given we inject script to each frame, we don't get much of "memory re-use" benefits. So, just here from a theoretical standpoint.

Added to class to keep it _scoped to_ class.

Reply:
Like you mentioned, since we inject the script in each frame, I would suggest to just make them privatem, not static. This way we avoid doing `FrameManager.handledMessage` and we can just use `this`, which is much shorter.